test/e2e/create_test.go

```diff
@@ -2,17 +2,15 @@ package e2e
 
 import (
 	"context"
-	"log"
-	"testing"
-	"time"
-
 	operator "github.com/openshift/windows-machine-config-operator/pkg/apis/wmc/v1alpha1"
 	wmc "github.com/openshift/windows-machine-config-operator/pkg/controller/windowsmachineconfig"
 	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	v1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/client-go/kubernetes"
+	"log"
```

Comment:
note: Change this to respect the split

Reply:
I am assuming you will make this change as part of this PR